states/pause.lua

```diff
@@ -35,10 +48,24 @@ end
 function pause:keypressed(key, scancode, isrepeat)
 	-- Return to the game
 	if key == "escape" then
-		nextState = self.gameState
+		self:resume()
```

Comment:
Good for now. But we might want to add a conditional here that if you're in the "options" part of the menu, you'll go back to the normal pause menu